app/src/services/subscribers/CensusEventSubscriberService.ts

```diff
@@ -119,18 +118,17 @@ export default class CensusEventSubscriberService implements ServiceInterface {
         });
     }
 
-    private checkAlert(event: PS2ZoneEvents): boolean {
+    private alertExists(worldId: string, zoneId: string): boolean {
```

Comment:
This may as well call the `ActiveAlertAuthority.checkExists()` function directly and then convert that to use world and zone, which I think it already may do, then delete this proxy function.